### DISCUSSION TITLE
DietPi-Software | Overseerr

### DISCUSSION THREAD

**Discussion by HDStremmie** [MichaIng/DietPi#5051] [Software Request]
#### Formal software information
- Software title | Overseerr 
- Short description | Beautiful Media Discovery / Simple Request Management
- Official URL (if available) | https://overseerr.dev/
- GitHub URL (if available) | https://github.com/sct/overseerr
- (Official) install documentation | https://docs.overseerr.dev/getting-started/installation

#### Are there similar/alternative software titles available with DietPi-Software?

- ...

#### What makes your requested software better than the above solutions, if available?
Maybe a little like Sonarr & Radarr. However these applications don’t support ‘Requests’. Overseerr integrates great with Sonarr, Radarr & Plex
- ...

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
It could be easy installed through ‘Dietpi-Software’ (all in one place) 
It’s installed through ‘snap’ not APT
- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
EG:
1. Just Install the software (integration with Radarr & Sonnarr could be easily done afterwards)
2. Maybe change Port of it conflicts with any other (of possible)

1. ...

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.

- ...

**Comment by Joulinar** [comment]
We are moving to GitHub discussion to collect software and image request in future. Therefore `issue` will be closed and reopen as `discussion.`
